crates/noirc_frontend/src/ast/expression.rs

```diff
@@ -244,6 +246,8 @@ impl BinaryOpKind {
             BinaryOpKind::Or => "|",
             BinaryOpKind::Xor => "^",
             BinaryOpKind::Assign => "=",
+            BinaryOpKind::Shr => ">>",
+            BinaryOpKind::Shl => "<<",
```

Comment:
Can we expand the name of these operators to `ShiftLeft` and `ShiftRight`, the other op kinds aren't abbreviated

Reply:
Done